src/app/store/flyer/[store_id]/page.tsx

```
@@ -0,0 +1,338 @@
+  price_including_tax: number;
+  unit: string;
+  restriction_note: string;
+}
+
+interface Campaign {
+  name: string;
+  start_date: string;
+  end_date: string;
+}
+
+interface Store {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'Store'. Consider to delete it or rename it.

'Store' is defined here:


(lint/suspicious/noRedeclare)